climpred/classes.py

```diff
@@ -55,7 +56,7 @@ def _check_reference_dimensions(init, ref):
     if 'member' in init_dims:
         init_dims.remove('member')
     if not (set(ref.dims) == set(init_dims)):
-        raise ValueError("""Dimensions must match initialized
+        raise DimensionError("""Dimensions must match initialized
             prediction ensemble dimensions (excluding `lead` and `member`.)""")
```

Comment:
I don't know if we want to loosen this up for multiple models.

Reply:
I think we should require that dims match. The user should do those before. Maybe we put the check only before we compute skill and not upon adding to the prediction object?

Reply:
I think we should check at the first entry into the pipeline. I.e., check when they add the simulation/obs to the prediction object. Then you never need to check the dimensions again. Since we expect `init`, `lead`, etc. why wouldn't we just check on the start?

Reply:
for now I agree. my comment was about later when we want to change hindcast and reference at the same time by the same function (eg. some regridding). Then we should allow different dimensions...

example flow then:
1. add your hindcast
2. add some downlaoded satellite reference with different spatial dims (not temporal)
3. regrid to the same 1x1 grid.

Reply:
Okay, we'll keep this in mind for v2 then.